safe_transaction_service/history/tests/test_views.py

```diff
@@ -2259,6 +2277,9 @@ def test_incoming_transfers_view(self):
 
         token_value = 6
         ethereum_erc_20_event = ERC20TransferFactory(to=safe_address, value=token_value)
+        ethereum_erc_20_event.generated_id = self._get_generated_id(
```

Comment:
Either you hardcode the expected result (by adding arguments to ERC20TransferFactory so it's deterministic) or you check that it follows the expected the schema, but copying the function to the tests and doing this here does not look right, same for the rest of the tests